### PR TITLE
Tooltip in <body>

### PR DESCRIPTION
**Pull request by latata**
Render tooltip in `<body>`, so it will be always visible independently from overflow settings of editor wrapper elements. 

@hakubo @xkxd

### CODE REVIEW

File: src/components/Tooltip.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	box-shadow: 0 0 20px 0 rgba(26, 26, 26, .3);
 	padding: 12px;
 	position: absolute;
-	z-index: 2;
+	z-index: 1000;
```
Comment (xkxd) [inline src/components/Tooltip.scss:7]: is it necessary to increase it that much?

File: src/components/HyperlinkingWrapper.js [hunk 2]
```diff
@@ -169,23 +167,21 @@ export default class HyperlinkingWrapper extends Component {
...
 
         return Math.min(defaultWidth, maxWidth);
     }
 
     getComputedTop(bottom) {
-        const { offsetTop } = this.quill.root.parentElement;
+        const { postEditorWrapper } = this.props;
 
-        return bottom + NOTCH_COMPENSATION + offsetTop;
+        return bottom + NOTCH_COMPENSATION + postEditorWrapper.getBoundingClientRect().top + window.scrollY;
```
Comment (xkxd) [inline src/components/HyperlinkingWrapper.js]: can you assign `postEditorWrapper.getBoundingClientRect().top + window.scrollY;` to `offsetTop` const?

File: src/components/HyperlinkingWrapper.js [hunk 3]
```diff
@@ -198,10 +194,10 @@ export default class HyperlinkingWrapper extends Component {
     }
 
     getComputedPosition(position) {
-        const { offsetLeft } = this.quill.root.parentElement;
+        const { postEditorWrapper } = this.props;
         const centerOfSelection = (position.left + position.right) / 2;
         const width = this.getComputedWidth();
-        const defaultLeft = centerOfSelection - width / 2 + offsetLeft;
+        const defaultLeft = centerOfSelection - width / 2 + postEditorWrapper.getBoundingClientRect().left + window.scrollX;
```
Comment (xkxd) [inline src/components/HyperlinkingWrapper.js]: same here, it'll be easier to understand

File: src/components/Tooltip.scss [hunk 4]
```diff
@@ -19,3 +19,8 @@
 		width: 0;
 	}
 }
+
+.pe-tooltip,
+.pe-tooltip * {
+	box-sizing: border-box;
```
Comment (hakubo) [inline src/components/Tooltip.scss:25]: just sharing:
in mobile-wiki there is
https://github.com/Wikia/mobile-wiki/blob/dev/app/styles/base/_box-sizing.scss
this is a nice default that is easily overridden when needed

File: src/components/Tooltip.scss (hunk 1 above)
Comment (latata) [reply src/components/Tooltip.scss:7]: yes

**Review by xkxd — APPROVED** [review]

**Pull request merged**